cluster/examples/kubernetes/ceph/cluster-on-pvc.yaml

```diff
@@ -91,6 +91,9 @@ spec:
       volumeClaimTemplates:
       - metadata:
           name: data
+          # if you are looking at giving your OSD a different CRUSH device class than the one detected by Ceph
+          # annotations:
```

Comment:
Should we move this annotation down to the metadata device template? That's where the common case would be for this setting, right?

Reply:
Yes but still you want to apply it on the "data" device because it's actually the OSD, the "metadata" device is just a backing device.

Reply:
Ok I see now. It's an OSD-level setting so we just look at it on the data template.